src/Primitives.hs

```diff
@@ -303,9 +304,11 @@ primitiveInfo _ ctx [target@(XObj (Sym path@(SymPath _ _) _) _ _)] = do
               <|> (multiLookupBinderEverywhere ctx path)
           )
     _ ->
-      case lookupBinderInContextEnv ctx path of
-        Nothing -> notFound ctx target path
-        Just found -> printer found
+      (printIfFound (lookupBinderInTypeEnv ctx path))
+        >> ( case lookupBinderInContextEnv ctx path of
+               Nothing -> notFound ctx target path
+               Just found -> printer found
+           )
   where
     printIfFound :: Maybe Binder -> IO (Context, Either EvalError XObj)
```

Comment:
Perhaps not related to this PR, but shouldn't this return `IO ()`?

Reply:
Definitely! Added a TODO about it: d963569